code/modules/mob/living/simple_animal/hostile/bees.dm

```diff
@@ -48,7 +48,46 @@
 	var/isqueen = FALSE
 	var/icon_base = "bee"
 	var/static/list/bee_icons = list()
-
+	
+	var/transfer_type = INJECT
+	var/transfer_min = 1
+	var/transfer_max = 5
+
+/mob/living/simple_animal/hostile/poison/bees/vapor//wave
+	name = "vaporizing bee"
+	desc = "You don't understand, it's all in your hands"
+	transfer_type = VAPOR//WAVE
+
+/mob/living/simple_animal/hostile/poison/bees/touch
+	name = "fuzzy bee"
+	desc = "You think touching that is a bad idea."
+	transfer_type = TOUCH
+
+/mob/living/simple_animal/hostile/poison/bees/highcap
+	name = "high-capacity bee"
+	desc = "This bee has an engorged sack. Looks kinda weak, though."
+	maxHealth = 5
+	health = 5
+	transfer_min = 5
+	transfer_max = 10
+	
+/mob/living/simple_animal/hostile/poison/bees/strong
+	name = "big bee"
+	desc = "This bee has been lifting lately.."
```

Comment:
whoever sprites this, I demand this bee the sprite in honor of the fallen goonswarm federation